code/modules/mob/living/carbon/human/species_types/lizardpeople.dm

```diff
@@ -161,6 +161,64 @@
 	. = ..()
 	C.weather_immunities -= "ash"
 
+//Ash walker shaman, worse defensive stats, but better at surgery and have a healing touch ability
+/datum/species/lizard/ashwalker/shaman
+	name = "Ash Walker Shaman"
+	id = "ashlizardshaman"
+	armor = -1 //more of a support than a standard ashwalker, don't get hit
+	brutemod = 1.15
+	burnmod = 1.15
+	speedmod = -0.1 //similar to ethereals, should help with saving others
+	punchdamagehigh = 7
+	punchstunthreshold = 7
+	action_speed_coefficient = 0.9 //they're smart and efficient unlike other lizards
+	var/obj/effect/proc_holder/spell/targeted/touch/healtouch/goodtouch
+
+//gives the heal spell
+/datum/species/lizard/ashwalker/shaman/on_species_gain(mob/living/carbon/C, datum/species/old_species)
+	. = ..()	
+	goodtouch = new /obj/effect/proc_holder/spell/targeted/touch/healtouch
+	C.AddSpell(goodtouch)
+
+//removes the heal spell
+/datum/species/lizard/ashwalker/shaman/on_species_loss(mob/living/carbon/C)
+	. = ..()
+	if(goodtouch)
+		C.RemoveSpell(goodtouch)
+
+//basic touch ability that heals brute and burn, only accessed by the ashwalker shaman
+/obj/effect/proc_holder/spell/targeted/touch/healtouch
+	name = "Healing touch"
+	desc = "This spell charges your hand with vile energy that can be used to violently explode victims with healing."
```

Comment:
```suggestion
	desc = "This spell charges your hand with the vile energy of the Necropolis, permitting you to undo some external injuries from a target."
```

Reply:
this was literally copy pasted from disintegration desc and then "with healing" tacked on, half expected you to replace the entire line